release-notes/v2.0.0.md

```diff
@@ -10,6 +10,16 @@
 
 The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.0.0.
 
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|BZ-134925|csi.k8s|Disable NodeGetVolumeStats capability|Set the parameter `disableNodeGetVolumeStats` to true at the time of deployment.|
+
+The following table lists the resolved issues for HPE Primera/3PAR CSP for HPE CSI Driver release 2.0.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|BZ-134504|csi.k8s| More online FC ports caused huge time for mount| New parameter `fcPortsList` is added in Storageclass and will be applicable only when a new volume is created.|
```

Comment:
Consider rewording as below:
|BZ-134504|csi.k8s| Suboptimal attach/detach performance when all FC ports are auto-discovered and used for multipathing| For a higher end primera array, there could be several FC ports available for use with multipathing. By default the CSP auto-discovers all available FC ports and uses them for multipathing. This increases the time taken for attach/detach operations significantly. To address this, a new parameter `fcPortsList` is added in Storageclass. This will be applicable only for newer volumes that are created.|

Reply:
done